pymeasure/display/console.py

```diff
@@ -0,0 +1,281 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2021 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+
+import os
+import copy
+import argparse
+try:
+    import progressbar
```

Comment:
This snagged my attention in passing: I haven't checked what that package does specifically, but in the last years, whenever the matter was including "progress bars" in Python, `tqdm` came up first and foremost (in articles, blogs, etc): https://tqdm.github.io/ What was the reason you chose `progressbar`?
(To be fully clear I have used neither up to now)

Reply:
> This snagged my attention in passing: I haven't checked what that package does specifically, but in the last years, whenever the matter was including "progress bars" in Python, `tqdm` came up first and foremost (in articles, blogs, etc): https://tqdm.github.io/ What was the reason you chose `progressbar`? (To be fully clear I have used neither up to now)

@bilderbuchi, thanks for having a look. Indeed, I did some google search and using progressbar2 I had good visual results. If there is no strong reason, I would leave this optional package instead of moving to tqdm.

Reply:
Fair enough, ok with me.